ui/lib/core/addon/helpers/changelog-url-for.js

```diff
@@ -0,0 +1,37 @@
+import { helper } from '@ember/component/helper';
+
+/*
+This helper returns a url to the changelog for the specified version.
+It assumes that Changelog headers for Vault versions >= 1.4.3 are structured as:
+
+## v1.5.0
+### Month, DD, YYYY
+
+## v1.4.5
+### Month, DD, YYY
+
+etc.
+*/
+
+export function changelogUrlFor([version]) {
+  let url = 'http://www.github.com/hashicorp/vault/blob/master/CHANGELOG.md#';
+
+  try {
+    // strip the '+prem' from enterprise versions and remove periods
+    let versionNumber = version
```

Comment:
I have no idea what all the various versions look like from the perspective of the UI (like the HSM ones). So, I can't comment if this will work in all cases, which is my only concern.

Reply:
yeah, we're not totally sure what all of the versions look like either which is why i wrapped this in a try block. this way, if we encounter any versions that don't meet the expected format we'll catch the error log the version. if anything fails we default to the regular 'ol changelog which should prevent users from having to deal with broken links.